collect/src/marinade_service.rs

```diff
@@ -117,18 +119,16 @@ fn get_stake_accounts(
         delegation_authority
     );
 
-    let mut filters = vec![RpcFilterType::Memcmp(Memcmp {
-        offset: 4 + 8, // enum StakeState + rent_exempt_reserve: u64
-        bytes: MemcmpEncodedBytes::Base58(delegation_authority.to_string()),
-        encoding: None,
-    })];
+    let mut filters = vec![RpcFilterType::Memcmp(Memcmp::new(
+        4 + 8,
+        MemcmpEncodedBytes::Base58(delegation_authority.to_string()),
+    ))];
 
     if let Some(withdrawer_authority) = withdrawer_authority {
-        filters.push(RpcFilterType::Memcmp(Memcmp {
-            offset: 4 + 8 + 32, // enum StakeState + rent_exempt_reserve: u64 + delegation_authority: Pubkey
-            bytes: MemcmpEncodedBytes::Base58(withdrawer_authority.to_string()),
-            encoding: None,
-        }));
+        filters.push(RpcFilterType::Memcmp(Memcmp::new(
+            4 + 8 + 32,
```

Comment:
Why were the comments removed? Were they wrong or misleading?

Reply:
Did not even notice the comments... These numbers are so common now for me :D Putting the comments back